app/onepassword_events_api/appserver/static/javascript/components/wizard.js

```diff
@@ -0,0 +1,179 @@
+import React from "react";
+
+const VERSION = "1.6.0";
+
+const e = React.createElement;
+export default class Wizard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentStep: 1,
+      totalSteps: props.steps.length,
+      success: false,
+      error: "",
+      authToken: "",
+    };
+  }
+
+  handleNext = async (event) => {
+    event.preventDefault();
+    this.setState({
+      currentStep: this.state.currentStep + 1,
+    });
+  };
+
+  handlePrev = async (event) => {
+    event.preventDefault();
+    this.setState({
+      currentStep: this.state.currentStep - 1,
+      success: false,
+      error: "",
+    });
+  };
+
+  handleSkip = async (event) => {
+    event.preventDefault();
+    this.setState({
+      currentStep: this.state.totalSteps,
+    });
+  };
+
+  handleSubmit = async (event) => {
+    event.preventDefault();
+    const result = await this.props.handleSubmit(this.state.authToken);
+    this.setState(result);
+  };
+
+  render() {
+    const stepDetails = this.props.steps[this.state.currentStep - 1];
```

Comment:
Let's add a little comment above that we are using 1 based indexing for steps and that's why we need to minus 1 to get the actual current step. At first I thought you were getting the previous step.